app/templates/todos/index.plush.html

```diff
@@ -24,41 +24,38 @@
         </div>
         <div class="px-5">
             <table class="table">
-                <thead class="thead">
+                <thead>
                 <tr class="text-white">
                     <th scope="col">Task</th>
                     <th scope="col">Complete By</th>
                     <th scope="col">Actions</th>
                 </tr>
                 </thead>
-                <tbody class="table-border" style="background-color: white;">
-                <tr>
-                    <th scope="row" class="task-color"><i class="fas fa-check-circle mr-3 gray-text"></i>Client Meeting</th>
-                    <td>08 May 2019</td>
-                    <td><i class="fas fa-pencil-alt mr-3 gray-text"></i><i class="fas fa-trash-alt red-text"></i></td>
-                </tr>
-                <tr>
-                    <th scope="row" class="task-color"><i class="fas fa-check-circle mr-3 gray-text"></i>Design App Interface</th>
-                    <td>15 May 2019</td>
-                    <td><i class="fas fa-pencil-alt mr-3 gray-text"></i><i class="fas fa-trash-alt red-text"></i></td>
-                </tr>
-                <tr>
-                    <th scope="row" class="task-color"><i class="fas fa-check-circle mr-3 gray-text"></i>Learn Node.Js</th>
-                    <td>03 Sep 2019</td>
-                    <td><i class="fas fa-pencil-alt mr-3 gray-text"></i><i class="fas fa-trash-alt red-text"></i></td>
-                </tr>
-                <tr>
-                    <th scope="row" class="task-color"><i class="fas fa-check-circle mr-3 gray-text"></i>Bring the puppy out XD</th>
-                    <td>24 Jul 2019</td>
-                    <td><i class="fas fa-pencil-alt mr-3 gray-text"></i><i class="fas fa-trash-alt red-text"></i></td>
-                </tr>
+                <tbody>
+                <%= for (index, value) in todos { %>
+                    <tr>
+                        <th scope="row" class="task-color"><i class="fas fa-check-circle mr-3 gray-text"></i><%= todos[index].Title %></th>
```

Comment:
th should not be used for displaying table data. This tag acts as table header only.
https://www.w3schools.com/tags/tag_th.asp

Also, when you use this syntax
`(index, todo) in todos`
Then you can access a todo using:
`todo.Title` instead of `todos[index].Title`
This is the convention we follow for plush iteration.